### DISCUSSION TITLE
Getting rid of these dependencies

### DISCUSSION THREAD

**Discussion by kairan77** [visual-space/visual-editor#80] [Feature Proposals]
I suggest get rid of all these dependencies

flutter_colorpicker: ^1.0.3
image_picker: ^0.8.5
photo_view: ^0.13.0
url_launcher: ^6.1.0
video_player: ^2.4.0
youtube_player_flutter: ^8.0.0
i18n_extension: ^4.2.1
gallery_saver: ^2.3.2

these are half-baked solutions which add more weight than value, they are not production ready, ugly and won't fit to spec for anyone's design, 

instead leave functional hooks open and give instructions to users on how to import them on their own, if they choose to do so.

**Comment by adrian-moisa** [comment]
Thanks for opening this ticket. It will take some time until we can go trough all of them. One concern that I have is that maybe we should have a hybrid mode. Where for example: for beginner low skill users we provide defaults, for advanced users we provide hooks. Any thoughts? Upgrading to better libs is also an option.
